doc/release-notes/8129-harvesting.md

```diff
@@ -0,0 +1,11 @@
+### Remap oai_dc export and harvesting format fields: dc:type, dc:date, and dc:rights
+
+The `oai_dc` export and harvesting format has had the following fields remapped:
+
+- dc:type was mapped to the field "Kind of Data". Now it is hard-coded to the word "Dataset".
+- dc:date was mapped to the field "Production Date" when available and otherwise to "Publication Date". Now it is mapped the field “Publication Date” or the field used for the citation date, if set (see [Set Citation Date Field Type for a Dataset](https://guides.dataverse.org/en/6.3/api/native-api.html#set-citation-date-field-type-for-a-dataset)).
+- dc:rights was not mapped to anything. Now it is mapped (when available) to terms of use, restrictions, and license.
+
+As these are backward incompatible changes, they have been noted in the [API changelog](https://guides.dataverse.org/en/latest/api/changelog.html).
```

Comment:
The PR looks great. But I'm not entirely sure if this needs to be in the API log. I feel like it would be enough to mention this in the release note, and recommend running a re-export - which seems to be our normal approach when making changes to metadata exports - ?

Reply:
Yeah, maybe you're right. I consider OAI-PMH an API but I went ahead and removed what I wrote in the API Changelog.

Hopefully, people will notice the backward incompatible changes in the release notes. I reworked the snippet a bit to emphasize this and also added a bit about reexport.